core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -563,8 +563,15 @@ class GroupCoordinator(val brokerId: Int,
     info(s"Removed $offsetsRemoved offsets associated with deleted partitions: ${topicPartitions.mkString(", ")}.")
   }
 
-  private def validateGroup(groupId: String): Option[Errors] = {
-    if (!validGroupId(groupId))
+  private def validateNonEmptyGroupStatus(groupId: String): Option[Errors] = {
```

Comment:
Can we add a comment saying that this is only used by `commit` and `fetch` and why?